.travis.yml

```diff
@@ -66,6 +66,10 @@ jobs:
       dist: trusty
       python: "2.7" # The old Android build system only supports python2
       script: .travis/script.sh
+    - env: BUILD_TARGET="gn-build" VERBOSE=1
+      os: linux
+      dist: trusty
```

Comment:
nit: is trusty required? if not, let's leave this out and use the global default (currently bionic).

Reply:
Done

Reply:
I still see trusty specified here. Can you make sure you pushed the change?

Reply:
sorry about that, I pushed the correct file today morning.